extensions/wikia/UserProfilePageV3/models/FavoriteWikisModel.class.php

```diff
@@ -27,11 +27,13 @@ class FavoriteWikisModel extends WikiaModel {
 	 * @param User $user
 	 */
 	public function __construct( User $user ) {
+		parent::__construct();
```

Comment:
I think its good form unless there is a specific reason not to.  Additionally, I often see code that does:
```php
$wg = \F::app()->wg;
```
in classes that would give them that variable if they'd just inherited from parent.